spring-websocket/src/main/java/org/springframework/web/socket/messaging/DefaultSimpUserRegistry.java

```diff
@@ -110,7 +113,9 @@ else if (event instanceof SessionConnectedEvent) {
 			if (user instanceof DestinationUserNameProvider destinationUserNameProvider) {
 				name = destinationUserNameProvider.getDestinationUserName();
 			}
-			synchronized (this.sessionLock) {
+
+			this.sessionLock.lock();
```

Comment:
```suggestion
			sessionLock.lock();
```
There is no overloaded field that makes the `this.` access required, right? Both are correct, but if there is no ambiguity, it's more concise to leave it out.

Reply:
Thank you for your valuable feedback.
I actually considered this when making the code changes.

Looking at other classes that use ReentrantLock, I noticed that they all include `this.`
so I decided to keep it for consistency. 
(e.g., `SubProtocolWebSocketHandler`, `InMemoryWebSessionStore.ExpiredSessionChecker`)